glide.yaml

```diff
@@ -0,0 +1,19 @@
+package: github.com/Workiva/go-rest
+import:
+- package: github.com/gorilla/context
+  version: ~1.1.1
+- package: github.com/gorilla/mux
+  version: ~1.6.2
+- package: github.com/blakeroberts-wk/mustache
```

Comment:
I'm not sure what the rules around pulling from our forks are, we should look into that

Reply:
Waiting on a Workiva fork: https://jira.atl.workiva.net/browse/COREAPPS-2112